bentoml/_internal/bento/bento.py

```diff
@@ -373,14 +381,66 @@ def __init__(self, base_path: t.Union[PathType, "FS"]):
         super().__init__(base_path, Bento)
 
 
+@attr.define
+class RunnerInfo:
+    name: str
+    runner_type: str
+    model_runner_module: t.Optional[str]
+
+    @classmethod
+    def from_runner(cls, r: "t.Union[Runner, SimpleRunner]") -> "RunnerInfo":
+        from ..frameworks.common.model_runner import BaseModelRunner
+
+        model_runner_module = (
+            r.model_info.info.module if isinstance(r, BaseModelRunner) else None
+        )
+        return cls(
+            name=r.name,  # type: ignore
+            runner_type=r.__class__.__name__,
+            model_runner_module=model_runner_module,
+        )
+
+
+@attr.define
+class ApiInfo:
+    name: str
+    input_type: str
+    output_type: str
+
+    @classmethod
+    def from_inference_api(cls, api: "InferenceAPI") -> "ApiInfo":
+        return cls(
+            name=api.name,
+            input_type=api.input.__class__.__name__,
+            output_type=api.output.__class__.__name__,
+        )
+
+
+@attr.define
+class ModelInfo:
```

Comment:
Maybe we should rename this because it's confusing to have this and `models.ModelInfo` be different things?

Reply:
changing to `BentoModelInfo`